homeworks/iryna.gychka_irina-hychka/homework_5/task_1.js

```diff
@@ -0,0 +1,75 @@
+// Task 1.1 Converts a numeric array to array of sums
+const xArray = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+const yArray = [0, 0, 0, 0, 0];
+const zArray = [10, -10, 10, -10, 10];
+
+function sumArray(n) {
+    return n.map((v, i) => (n.slice(0, i + 1).reduce((sum, el) => (sum + el))));
```

Comment:
once again, no need to `slice` and `reduce`... use closure mechanism and single `map()` operation